packages/edit-site/src/components/sidebar-edit-mode/index.js

```diff
@@ -85,14 +85,18 @@ export function SidebarComplementaryAreaFills() {
 		hasBlockSelection,
 		supportsGlobalStyles,
 		isEditingPage,
+		isEditorOpen,
 	} = useSelect( ( select ) => {
 		const _sidebar =
 			select( interfaceStore ).getActiveComplementaryArea( STORE_NAME );
+
 		const _isEditorSidebarOpened = [
 			SIDEBAR_BLOCK,
 			SIDEBAR_TEMPLATE,
 		].includes( _sidebar );
 		const settings = select( editSiteStore ).getSettings();
+		const { getCanvasMode } = unlock( select( editSiteStore ) );
```

Comment:
`select( editSiteStore )` is also invoked on the previous line — we should probably refactor the code to invoke it only once

Reply:
Yeah... somehow I convinced myself I needed to separate them because of the `unlock` call, but I don't actually. Fixed, thank you!